instatag.module

```diff
@@ -130,3 +130,10 @@ function instatag_published_revision_state($node) {
   }
   return FALSE;
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter.
+ */
+function instatag_form_instatag_settings_alter(&$form, &$form_state, $form_id) {
+  $form['instatag_snippet_url']['#maxlength'] = 512;
```

Comment:
@abopche ,
Generally hook_form_FORM_ID_alter is meant for altering form defined by other modules. Here we can directly edit `instatag_settings()` callback to get desired effect. Check `instatag.admin.inc`

Also update the variable name from `instatag_snippet_url` -> `instatag_snippet`.
Please change the form type `textfield` to `textarea` too, in which case the maxlength will become redundant.